tests/backends/test_pyevm.py

```diff
@@ -24,4 +23,31 @@ def eth_tester():
 
 
 class TestPyEVMBackendDirect(BaseTestBackendDirect):
-    pass
+
+    def test_override_genesis_parameters(self, eth_tester):
+        super().test_reset_to_genesis(eth_tester)
```

Comment:
It strikes me as odd to run a test directly, especially inside another test. Maybe make a helper function that does a genesis reset, which both tests call.